pkgs/by-name/gi/git-identity/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  ronn,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "git-identity";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "madx";
+    repo = "git-identity";
+    rev = "v${version}";
+    hash = "sha256-u4lIW0bntaKrVUwodXZ8ZwWxSZtLuhVSUAbIj8jjcLw=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+    ${lib.getExe ronn} --roff git-identity.1.ronn
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin $out/share/man/man1 $out/share/bash-completion/completions $out/share/zsh/site-functions
+    cp git-identity $out/bin/git-identity
+    cp git-identity.1 $out/share/man/man1/
+    cp git-identity.bash-completion $out/share/bash-completion/completions/git-identity-completion.bash
+    cp git-identity.zsh-completion $out/share/zsh/site-functions/_git-identity
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Manage your identity in Git";
+    mainProgram = "git-identity";
+    homepage = "https://github.com/madx/git-identity";
+    license = licenses.wtfpl;
+    maintainers = with maintainers; [ mynacol ];
+    platforms = platforms.all;
+  };
```

```suggestion
  meta = {
    description = "Manage your identity in Git";
    mainProgram = "git-identity";
    homepage = "https://github.com/madx/git-identity";
    license = lib.licenses.wtfpl;
    maintainers = with lib.maintainers; [ mynacol ];
    platforms = lib.platforms.all;
  };
```

Usage of `with lib;` is discouraged.